auth0/structure_auth0_connection.go

```diff
@@ -116,6 +116,25 @@ func flattenConnectionOptionsSMS(o *management.ConnectionOptionsSMS) interface{}
 	}
 }
 
+func flattenConnectionOptionsOIDC(d Data, o *management.ConnectionOptionsOIDC) interface{} {
```

Comment:
I think we don’t need the Data argument here right?

Reply:
You are right!